### ISSUE TITLE
Slack Link in github Issue Template is invalid

### ISSUE THREAD

**Issue by mehrad-rafigh** [nrwl/nx#3538]
The following line needs to be updated
https://github.com/nrwl/nx/blob/a4d0f11e737dbc3981d8414656b8554e1e17f7ef/.github/ISSUE_TEMPLATE/config.yml#L7

**Comment by FrozenPandaz** [comment]
Good catch! Would someone from the community like to send a PR to use the link in the README.md instead? That one is still valid.
